scripts/codegen-cli.js

```diff
@@ -4,7 +4,7 @@ const {APIClient} = require('@wharfkit/session')
 const fs = require('fs')
 const {fetch} = require('node-fetch')
 
-const {codegen, Contract} = require('../lib/contract.js')
+const {codegen, ContractKit} = require('../lib/contract.js')
 
 const client = new APIClient({
     url: process.env.ANTELOPE_NODE_URL || 'https://eos.greymass.com',
```

Comment:
Can we make this a parameter to the call itself? 

We should do one other the other:

1. Allow passing in all params to the script (like we can with `wharf-generate {contractName}`)
2. Make all params environmental variables (`ANTELOPE_NODE_URL=something ANTELOPE_CONTRACTNAME=something wharf-generate`)

But probably not both at the same time.